.goreleaser.yml

```diff
@@ -20,7 +20,9 @@ builds:
             - QUILL_LOG_FILE=target/dist/quill-{{ .Target }}.log
     ignore:
       - goos: windows
-        goarch: arm64
+        goarch: arm
+      - goos: darwin
+        goarch: arm
```

Comment:
Given that `goarch: [ amd64, arm64 ]` what happens here with the arm change?
Besides that I do not know why we would like not to build arm image @Jeffail?